test/broadcast.jl

```diff
@@ -609,4 +606,4 @@ end
 # end
 
 # Issue #22180
-@test isequal(convert.(Nullable, [1,2]), [Nullable(1), Nullable(2)])
+@test convert.(Any, [1, 2]) == [1, 2]
```

Comment:
@timholy Is this OK? I took the test from https://github.com/JuliaLang/julia/pull/22180, but I guess there was a reason why you changed it?